bundles/org.openhab.automation.jrubyscripting/src/main/resources/OH-INF/config/config.xml

```diff
@@ -102,5 +108,13 @@
 			<advanced>true</advanced>
 		</parameter>
 
+		<parameter name="console" type="text" required="false" groupName="console">
+			<label>Console Script</label>
+			<description><![CDATA[The script file to be required by the <tt>jrubyscripting console</tt>
+			console command.]]></description>
+			<default>openhab/console/irb</default>
+			<advanced>true</advanced>
```

Comment:
if the whole group is already advanced, does the individual item need other be advanced?

Reply:
I tried removing the advanced tag from the individual items, whilst keeping it in the group. This would make them show up even when the "advanced" checkbox isn't selected. It seems like a "bug" somewhere, but for the time being we'll have to stick with how it goes.